languages/tree-sitter-stack-graphs-javascript/src/stack-graphs.tsg

```diff
@@ -266,38 +331,27 @@ attribute node_symbol = node            => symbol = (source-text node), source_n
 ;; ### Program Queries
 
 (program)@prog {
-
-    node @prog.before_scope
-    node @prog.after_scope
-    node @prog.module_scope
-    node @prog.module_pop
-    node @prog.exports
-    node @prog.exports_pop
-    
     scan FILE_PATH {
         "([^/]+)\.js$" {
-        attr (@prog.module_pop) pop_symbol = $1
-        ;;   attr (@prog.module_pop) no_span
```

Comment:
I think we want this `no_span` to persist otherwise we get weird errors in Aleph?